app/views/my/password.html.erb

```diff
@@ -50,6 +48,7 @@ See doc/COPYRIGHT.rdoc for more details.
       <%= styled_label_tag 'password', User.human_attribute_name(:password) %>
       <div class="form--field-container">
         <%= styled_password_field_tag 'password', nil, :size => 25 %>
+        <span class="advanced-tooltip"></span>
```

Comment:
so, I will basically have to do this to __every__ field in a form whenever one field gets a tooltip?